app/Actions/Search/PhotoSearch.php

```diff
@@ -63,11 +48,8 @@ public function query(array $terms)
 		$photos = $query->get();
 
 		return $photos->map(
```

Comment:
I'm wondering, is the `->toArray()` really necessary ? :thinking: 
I would expect the cast to Json actually calling this method.

Reply:
Your're are right. I will change that. The method will directly return a `Collection`.